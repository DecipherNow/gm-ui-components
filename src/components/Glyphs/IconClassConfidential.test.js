import React from "react";
import { shallow } from "enzyme";
import IconClassConfidential from "./IconClassConfidential.svg";
describe("IconClassConfidential", () => {
  it("matches snapshot", () => {
    
    const aIconClassConfidential = shallow(<IconClassConfidential />).find("IconClassConfidential.svg").dive();
    console.log(aIconClassConfidential.debug())
    expect(aIconClassConfidential).toMatchSnapshot();
  });
});