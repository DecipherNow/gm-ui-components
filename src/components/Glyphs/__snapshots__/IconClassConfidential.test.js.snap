// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconClassConfidential matches snapshot 1`] = `<IconMock />`;
